libr/bin/p/xtac.mk

```diff
@@ -0,0 +1,27 @@
+# ####################################################################
+# Plugin for XTA cache file - Makefile
```

Comment:
Dont put copyright/license stuff in includes or makefiles. Only in the .c is enough